tests/test_bolt2-01-close_channel.py

```diff
@@ -105,7 +109,11 @@ def test_close_channel_shutdown_msg_normal_case_sender_side(runner: Runner) -> N
     # the option that the helper method feels for us
     test_opts = {}
     pre_events_conn = connect_to_node_helper(
-        runner, tx_spendable=tx_spendable, conn_privkey="03"
+        runner=runner,
+        tx_spendable=tx_spendable,
+        conn_privkey="03",
+        global_features=runner.runner_features(field_name="global_features"),
+        features=runner.runner_features(field_name="features"),
```

Comment:
I was thinking these are the required feature, so the init message should always have those.

Why we are not putting inside the `connect_to_node_helper`  so we avoid in this way to call every time this `runner.runner_features`

Reply:
Yeah that would be better! Do you want this `runner.runner_features` as default argument inside `connect_to_node_helper`?

Reply:
I guess that you can call `runner.runner_features` in both cases right?

when it is None, you call vanilla `runner.runner_features` otherwise you add an additional bitfield to `runner.runner_features`

Reply:
I think i should modify the default `runner.runner_features` to use the additional features as well. Right now it's just returning an empty string